<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wifi</class>
 <widget class="QWidget" name="Wifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
	
         
          <item row="0" column="1">
		
		<layout class="QVBoxLayout" name="verticalLayout_3">
		
        <item>
         <layout class="QGridLayout" name="gridLayout_3">



      <item row="0" column="0">
       <widget class="QPushButton" name="pbt_open">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>


      <item row="1" column="0">
       <widget class="QPushButton" name="pbt_scan">
        <property name="text">
         <string>scan</string>
        </property>
       </widget>
      </item>

     
      

      
      
        <item row="2" column="0">
           <widget class="QLabel" name="label">
	<property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
            <property name="text">
             <string>name:</string>
            </property>
           </widget>
          </item>


        <item row="3" column="0">
         <widget class="QLineEdit" name="Edit_name">
          <property name="text">
           <string/>
          </property>
         </widget>
      </item>

	 <item row="4" column="0">
           <widget class="QLabel" name="label_2">
	<property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
            <property name="text">
             <string>password:</string>
            </property>
           </widget>
          </item>

    
        <item row="5" column="0">
         <widget class="QLineEdit" name="Edit_pass">
          <property name="text">
           <string/>
          </property>
         </widget> 
      </item>

	<item row="6" column="0">
       <widget class="QPushButton" name="pbt_connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>

          
         </layout>
        </item>
       </layout>


          </item>
          
          
          <item row="0" column="0">
           <widget class="QTextEdit" name="Edit_show"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
